<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/settings.ico</normaloff>../Icons/settings.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #1E1E1E;
}
QPushButton{
	background-color: black;
	color: yellow;
	border: 1px solid #483D8B;
}
QPushButton:hover{
	border: 2px dashed #00FA9A;
	color: #00FA9A;
}
QPushButton:pressed{
	background-color: #2F4F4F;
	color: #FF0000;
	border: 1px dashed #000080;
}
QPushButton:disabled{
	background-color: black;
	color: #800000;
	border: none;
}

QCheckBox{
	color:#FF4500;
}
QCheckBox:hover{
	color: #00FF00;
	border: 1px dashed #00FF00;
}

QCheckBox::indicator {
    width: 25px;
    height: 25px;
    background-color: gray;
    border-radius: 10px;
    border-style: solid;
    border-width: 1px;
    border-color: white white black black;
}
QCheckBox::indicator:checked {
    background-color: qradialgradient(spread:pad, 
                            cx:0.5,
                            cy:0.5,
                            radius:0.9,
                            fx:0.5,
                            fy:0.5,
                            stop:0 rgba(0, 255, 0, 255), 
                            stop:1 rgba(0, 64, 0, 255));
}
QCheckBox:checked, QCheckBox::indicator:checked {
    border-color: black black white white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>59</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Зберегти</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>229</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Після запуску згорнути додаток</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FAQ_check_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>38</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FAQ_check_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>38</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>275</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Додати до автозавантаження Windows</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
